openassessment/xblock/static/js/spec/lms/oa_file_upload.js

```diff
@@ -36,7 +36,7 @@ describe("OpenAssessment.FileUploader", function() {
             async: false,
             processData: false,
             contentType: 'image/jpeg',
-            headers: { 'Content-Disposition': `attachment; filename="${TEST_FILE.name}"` },
+            headers: { 'Content-Disposition': `attachment; filename*=UTF-8''${encodeURI(TEST_FILE.name)}` },
```

Comment:
What's with the two single quotes between `UTF-8` and `${` ?

Reply:
^ It optionally includes language info (e.g. `utf-8'en'<filename>`) but can be omitted if we don't have that language meta

Reply:
I don't totally understand, but I understand enough to say this looks good to me